File: Java/Fizzbuzz.java
import java.util.Scanner;

public class Fizzbuzz {
	public static void main(String[] args) {
		System.out.println("Enter a number:");
		Scanner input = new Scanner(System.in);
		int BOUND = input.nextInt();
		fizzBuzz(BOUND);
		input.close();
	}

	public static void fizzBuzz(int bound) {
		for (int i = 1; i <= bound; i++) {
			boolean fizzOrBuzz = false;
			if (i % 3 == 0) {
				System.out.print("Fizz");
				fizzOrBuzz = true;
			}
			if (i % 5 == 0) {
				System.out.print("Buzz");
				fizzOrBuzz = true;
			}

			if (fizzOrBuzz) {
				System.out.println();
			} else {
				System.out.println(String.valueOf(i));
			}
		}
	}
}

File: Python/fizzbuzz.py
#!/usr/bin/env python3
# Write a program that prints the numbers from 1 to 100. 
# But for multiples of "three print 'Fizz' instead of the number 
# and for the multiples of five print 'Buzz'. 
# For numbers which are multiples of both three and five print 'FizzBuzz'.

print("Enter the Fizzbuzz limit:")
limit = int(input())

for x in range(1,limit+1): # this is range 1..15
	s = ""
	if x % 3 == 0:
		s += "Fizz"
	if x % 5 == 0:
		s += "Buzz"
	if s == "":
		s = x
	print (s)